internal/util/logging/logging.go

```diff
@@ -24,8 +24,30 @@ import (
 
 // Setup initializes logging with a given level.
 func Setup(level zapcore.Level) {
-	config := zap.NewDevelopmentConfig()
-	config.Level = zap.NewAtomicLevelAt(level)
+	var config zap.Config
+	if level <= zapcore.DebugLevel {
+		config = zap.Config{
+			Level:            zap.NewAtomicLevelAt(level),
+			Development:      true,
+			Encoding:         "console",
+			EncoderConfig:    zap.NewDevelopmentEncoderConfig(),
+			OutputPaths:      []string{"stderr"},
+			ErrorOutputPaths: []string{"stderr"},
+		}
+	} else {
+		config = zap.Config{
```

Comment:
For now, let's use the same configuration as for debug above except `Development` field value. We will change it later